### PR TITLE
Improve error messages.

### PR DESCRIPTION
**Pull request by floitsch**

### CODE REVIEW

File: pkg/tpkg/manager.go [hunk 1]
```diff
@@ -373,22 +373,22 @@ func (m *ProjectPkgManager) InstallLocalPkg(ctx context.Context, name string, pa
 		if isFile, err := isFile(target_spec); !isFile || err != nil {
 			if err == nil {
 				if !doesPathExist(target_spec) {
-					return "", m.ui.ReportError("Missing package.yaml in '%s'", path)
+					return "", m.ui.ReportError("Missing 'package.yaml' in '%s'", path)
 				}
-				return "", m.ui.ReportError("The package.yaml in '%s' is not a file", path)
+				return "", m.ui.ReportError("Couldn't read 'package.yaml' at '%s': not a file", path)
```
Comment (kasperl) [inline pkg/tpkg/manager.go:378]:
```suggestion
				return "", m.ui.ReportError("Cannot read 'package.yaml' at '%s': not a file", path)
```

File: pkg/tpkg/manager.go [hunk 2]
```diff
@@ -373,22 +373,22 @@ func (m *ProjectPkgManager) InstallLocalPkg(ctx context.Context, name string, pa
...
 			}
-			return "", m.ui.ReportError("The package.yaml in '%s' is not a file: %v", path, err)
+			return "", m.ui.ReportError("Couldn't read 'package.yaml' at '%s': %v", path, err)
```
Comment (kasperl) [inline pkg/tpkg/manager.go:380]:
```suggestion
			return "", m.ui.ReportError("Cannot read 'package.yaml' at '%s': %v", path, err)
```

**Review by kasperl — APPROVED** [review]
Looks good to me.

File: pkg/tpkg/manager.go (hunk 1 above)
Comment (floitsch) [reply pkg/tpkg/manager.go]: done.

File: pkg/tpkg/manager.go (hunk 2 above)
Comment (floitsch) [reply pkg/tpkg/manager.go]: done.

**Pull request merged**